README.md

```diff
@@ -1,3 +1,12 @@
+# Upcoming
+
+Cloud Storage FUSE V2.0 GA is round the corner and will be available on March
```

Comment:
I would put something like this: 

This file is ahead of what is released as the product is getting ready towards launching V2.0. 

To access the latest released version's document, please read: <put a link to what we will release in Feb 2024>

Reply:
ah thanks. We would do another merge later close to the V2.0 release I guess. 

The new text you proposed looks good.